presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -156,11 +156,31 @@ Unit              Description
 
     Returns ``timestamp2 - timestamp1`` expressed in terms of ``unit``.
 
-.. function:: parse_duration(string) ->  Interval
+Duration Function
+-----------------
+
+The ``parse_duration`` function supports the following units:
```

Comment:
This follows the convention that the table applies to the whole section. We do the same for `date_trunc`.